### PR TITLE
CONTRIBUTING: Mention not to open several pulls

### PR DESCRIPTION
**Pull request by luke-jr**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -87,6 +87,15 @@ before it will be merged. The basic squashing workflow is shown below.
     # save and quit
     git push -f # (force push to GitHub)
 
+If you need help with squashing (or other workflows with `git`), you can
+alternatively enable "Allow edits from maintainers" in the right GitHub
+sidebar and ask for help in the pull request.
```
Comment (luke-jr) [inline CONTRIBUTING.md]: Not sure this is something we want to encourage...

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -87,6 +87,15 @@ before it will be merged. The basic squashing workflow is shown below.
...
+
+Please refrain from creating several pull request for the same topic.
```
Comment (luke-jr) [inline CONTRIBUTING.md]: "same change"?

File: CONTRIBUTING.md (hunk 1 above)
Comment (laanwj) [inline CONTRIBUTING.md]: Well, not in general, but we want to encourage it for people that don't know how to use git. I'd prefer people to learn how to do it themselves, of course, but personally tutoring everyone how to use git (especially those that use the web interface :( ) is a lot of work.

**Review by maflcko — APPROVED** [review]
Addressed feedback

**Review by luke-jr — COMMENTED** [review]
ACK

File: CONTRIBUTING.md (hunk 2 above)
Comment (laanwj) [inline CONTRIBUTING.md]: should this be "several pull requests"?

**Review by jonasschnelli — APPROVED** [review]
ACK

**Comment by gmaxwell** [comment]
Seems reasonable, though the minor "Fix 0.13 release notes attribution mistake" commit seems unrelated.

**Pull request merged**